source/base/ReferenceCounter.ooc

```diff
@@ -22,6 +21,12 @@ ReferenceCounter: class {
 		this _lock = Mutex new(mutexType)
 //		"  RC, #{This count}, #{this _target class name}" println()
 	}
+	free: override func {
+		This count -= 1
+//		"--RC, #{This count}, #{this _target class name}" println()
```

Comment:
No that sounds like a good plan. I've just wanted to remove these comments for a long time, but keep forgetting it every time.